.github/workflows/sync-from-fork.yml

```diff
@@ -1,15 +1,17 @@
-name: Check for Changes
+name: Sync from Fork
```

Comment:
lowercase f? at least based on casing of `name` items below. or how does GitHub do for their actions?

Reply:
fixed